src/parser/test/ParserTest.cpp

```diff
@@ -1098,4 +1098,25 @@ TEST(Parser, ConfigOperation) {
     }
 }
 
+TEST(Parser, Limit) {
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 1 OVER work | LIMIT 1";
```

Comment:
since in 99% case, limit works together with order by, can you test them together, it's simple -- the parser test , I mean